test/test_linearsolvers.jl

```diff
@@ -6,8 +6,12 @@ using PythonCall
 @info("Test Solver on overdetermined system")
 Nobs = 10_000
 Nfeat = 100
-A = randn(Nobs, Nfeat) / sqrt(Nobs)
-y = randn(Nobs)
+A1 = randn(Nobs, Nfeat) / sqrt(Nobs)
+U, S1, V = svd(A)
```

Comment:
This line breaks the tests because of a typo. I would fix it myself, but I have a question. The changes in this block affect the tests for ALL solvers by redefining A to be something slightly different. I'm not opposed to this in principle, but are you sure it's what you want?

Reply:
I'll fix the typo, sorry I wasn't paying attention there.